plugins/datadog-entity-sync-backend/src/baseTransformExtension.ts

```diff
@@ -29,7 +29,7 @@ export const datadogServiceFromComponentSerializer = createBackendModule({
         datadogSync.defineSerializer({
           syncId: SYNC_ID,
           entityFilter,
-          serialize: defaultComponentSerializer,
+          serialize: entity => JSON.stringify(defaultEntitySerializer(entity)),
```

Comment:
If this is the prefered behavoir, I would say we should build this into how this functions
https://github.com/cvent/datadog-backstage-plugins/blob/1b9807524ee0b6ed26385d323081a0393c326c65/plugins/datadog-entity-sync-node/src/services/DatadogServiceFromEntitySync.ts#L146
https://github.com/cvent/datadog-backstage-plugins/blob/1b9807524ee0b6ed26385d323081a0393c326c65/plugins/datadog-entity-sync-node/src/services/DatadogServiceFromEntitySync.ts#L159-L161

We can either stringify it there or the `upsertCatalogEntity` might convert the `body` to a string internally anyway. The type could expect an Entity rather than a string too.